### ISSUE TITLE
TYPO3 8.7.x compatibility

### ISSUE THREAD

**Issue by SvenJuergens** [PHORAX/formhandler#3]
hi,

Do you have plans for a TYPO3 8.7.x release? and possible a TER release?

Best Regards

**Comment by mgrn0** [comment]
any updates on this question/issue? I tried, too and cannot even get a form to show up at all in the frontend. That's a bummer - formhandler was one of the few extensions we used in almost every project..

**Comment by klodeckl** [comment]
This would be interesting for me, too.

**Comment by 7elix** [comment]
Massive update available. Please test-drive. Comments are welcome.

**Comment by aw350me** [comment]
I just implemented the new formhandler version in a Typo3 8.7.8. 
Worked like a charm - had no problems at all. 

I required it from here https://packagist.org/packages/phorax/formhandler in version 3.0.0-beta1 and activated it in the extension manager afterwards. Then I just placed a new formhandler content element, selected the predefined form (which was still present from my old typo3 version) and saved. 
Frontend showed the form again then and I did test the email sending then. 
This also worked perfectly. 

Thank you very much 7elix, good job!
